modules/internal/ChapelTuple.chpl

```diff
@@ -293,6 +293,18 @@ module ChapelTuple {
     }
   }
 
+  /* TODO: Want this for heterogeneous tuples, but we can't write it today:
+
+  iter _tuple.indices param {
+    for param i in 1..this.size do
+      yield i;
+  }
+  */
+
+  proc _tuple.indices {
```

Comment:
We probably want an actual doc-string here (assuming this gets rendered in the docs)

Reply:
It doesn't.  In a follow-up commit, I've added spec text about tuple.indices.

Reply:
Will the field show up here? https://chapel-lang.org/docs/master/builtins/ChapelTuple.html

Reply:
Ah, thanks for pointing that out.  I was basing my analysis on not having seen a prototype for tuple.indices being generated there.  But I see how I can force it to now.